SVProfilePictureViewer/Language.cs

```diff
@@ -91,5 +91,23 @@ internal static class Language {
         };
       }
     }
+
+    internal static string WarnTitle {
```

Comment:
I would say remove this in favour of just using the PluginName as the title for the MessageBox.